packages/next-admin/src/utils/options.ts

```diff
@@ -0,0 +1,19 @@
+import { Field, ModelName, NextAdminOptions } from "../types";
+
+export const getCustomInputs = (
```

Comment:
Can you add a comment to explain what is doing this function ? Also can you add tests to all utils that you are adding.